pjmedia/src/pjmedia/transport_srtp_dtls.c

```diff
@@ -528,6 +528,9 @@ static void ssl_destroy(dtls_srtp *ds)
 	}
 	SSL_free(ds->ossl_ssl); /* this will also close BIOs */
 	ds->ossl_ssl = NULL;
+	/* thus reset the BIOs as well */
+	ds->ossl_rbio = NULL;
+	ds->ossl_wbio = NULL;
     }
```

Comment:
Shouldn't some check be added to ssl_on_recv_packet() as well?
i.e.
`if (ds->ossl_rbio == NULL) return some error (PJ_EGONE?)`

Reply:
This check is already available inside bio_write_intern
But I'll add it to PjSip as well to be independent